.ci/pipelines/build.Jenkinsfile

```diff
@@ -106,13 +106,24 @@ pipeline {
                     ],
                     wait: false
 
+                // test the latest version of OCP on every build
                 build job: 'cloud-on-k8s-e2e-tests-ocp',
                     parameters: [
                         string(name: 'JKS_PARAM_OPERATOR_IMAGE', value: operatorImage),
+                        string(name: 'OCP_VERSION', value: "4.8.5"),
                         string(name: 'branch_specifier', value: GIT_COMMIT)
                     ],
                     wait: false
 
+                // schedule another job for all the older 4.x OCP versions which runs only on Fridays (via when directive in job)
+                build job: 'cloud-on-k8s-e2e-tests-ocp-all-but-latest',
+                    parameters: [
+                        string(name: 'JKS_PARAM_OPERATOR_IMAGE', value: operatorImage),
+                        string(name: 'branch_specifier', value: GIT_COMMIT)
+                    ],
+                    quietPeriod: 86400, // add a 24 hour delay to this job to run it over the weekend
```

Comment:
IIUC we start the job on Friday but wait 24h before executing it. Why don't we start it on Saturday instead?

Reply:
Because I wanted to have it as a downstream job of the nightly build so that we do not have to build another operator image. In theory we could of course reuse the last snapshot build but I guess the image name would be a bit hard to discover from within the job when it is not passed in from upstream.